vault_core/src/config.rs

```diff
@@ -83,15 +85,15 @@ impl Config {
     }
 
     pub fn deposit_withdrawal_fee_cap_bps(&self) -> u16 {
-        self.deposit_withdrawal_fee_cap_bps.into()
+        u16::from(self.deposit_withdrawal_fee_cap_bps).min(MAX_BPS)
```

Comment:
imo worth removing these, kinda weird to translate in getters

Reply:
10-4